backuprestore.adoc

```diff
@@ -800,12 +799,14 @@ value.
 
 Three types of restore procedures can be run:
 
-The `zmrestore` command is used to restore the mailboxes while the
+* The `zmrestore` command is used to restore the mailboxes while the
 {product-name} mailbox server is running.
 
-* `zmrestoreoffline` -- to restore the mail server when the mail server is
-down. This command is run for disaster recovery.
-* `zmrestoreldap` -- to restore the content of the LDAP directory server.
+* The `zmrestoreoffline` command is used restore the mail server when
```

Comment:
nit: again inconsistent use of terminology (see zmrestore just before)
- perhaps remove the {product-name} before "mailbox server" on the zmrestore line?
- change "mail server" in this change to "mailbox server" for consistent use of terminology?